package.json

```diff
@@ -11,10 +11,12 @@
     "dist"
   ],
   "scripts": {
+    "commit": "northbrook commit",
```

Comment:
I generally use `northbrook` in my scripts to make it clearer what I'm doing. Will make them all as such if you want?